src/api/createTaskLineModal.ts

```diff
@@ -0,0 +1,46 @@
+import type { App } from 'obsidian';
+import type { Task } from '../Task';
+
+/**
+ * Interface to remove all references to {TaskModal} in this file.
+ * This is necessary to make {createTaskLineModal} testable.
+ * Once Jest is configured to work with Svelte, this can be removed.
+ */
+export interface ITaskModal {
+    open(): void;
+}
+
+/**
+ * Signature of the factory method for {TaskModal}.
```

Comment:
WebStorm does not recognise names just in `{}` as links.
It does work with `@link`, e.g. `{@link createTaskLineModal}`.

Reply:
Fixed.